rma_product_exchange/views/rma_operation_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+        <record id="rma_operation_tree" model="ir.ui.view">
+            <field name="model">rma.operation</field>
+            <field name="inherit_id" ref="rma.rma_operation_tree" />
+            <field name="arch" type="xml">
+                <field name="delivery_policy" position="after">
+                    <field name="product_exchange" optional="show" />
+                </field>
+            </field>
+        </record>
+
+        <record id="rma_operation_form" model="ir.ui.view">
+            <field name="model">rma.operation</field>
+            <field name="inherit_id" ref="rma.rma_operation_form" />
+            <field name="arch" type="xml">
+                <xpath expr="//group[@name='policies']" position="inside">
```

Comment:
It should be invisible if the delivery_policy is set to "no", because it is only used for delivery, no?